pkgs/test/test/runner/browser/runner_test.dart

```diff
@@ -2,8 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-@TestOn('vm')
+import 'dart:convert';
 
+@TestOn('vm')
```

Comment:
This annotation needs to move to the first import (or add a library tag and annotate that)